### PR TITLE
Impl of "show" command

### PR DESCRIPTION
**Pull request by matthiasbeyer**
This is ugly, monolithic, copy-pasta-code and does not even compile.

But I open this PR anyways for tracking.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -95,6 +96,175 @@ fn get_issue_tree_init_hashes(repo: &Repository, _: &clap::ArgMatches) -> i32 {
...
 
+/// show subcommand implementation
+fn show(repo: &Repository, sub: &clap::ArgMatches) -> i32 {
+    let issue = match sub.value_of("parent") {
+        Some(s) => s,
+        None => {
+            error!("No parent hash");
+            return 1;
+        },
+    };
```
Comment (neithernut) [inline src/main.rs]: If configured correctly, clap should generate an error. An `unwrap` should be sufficient.

**Comment by neithernut** [comment]
You could start feeding leaves of an issue to a `RevWalk` and plotting a simple graph using an iterator pattern. With a few tricks, you can transform a `RevWalk` to a lines iterator which gives you a graph to print.
You could then extend that tree printing iterator and have each item of that iterator expand to a message, including the graph portion at the beginning of the line.

**Comment by neithernut** [comment]
Please rebase and remove the bash version of the subcommand.

**Comment by matthiasbeyer** [comment]
As soon as this is in a shape where one can use it.

**Pull request closed without merging**